pages/home.jsx

```diff
@@ -146,11 +150,123 @@ var BlogSection = React.createClass({
+          <p className="error-msg">Please enter an email address.</p>
+        </div>
+      );
+    }
+  },
+  render: function() {
+    return (
+      <Modal modalTitle="" className="modal-pledge folded">
+        <ImageTag className="image center-block"
+                  src1x="/img/pages/home/svg/icon-teach-man-chalkboard-pledge.svg"
+                  alt="" width={150} height={150} />
+        <p>Because the Web is a global public resource that's integral to modern life, <strong>I pledge to:</strong></p>
+        <ul>
+          <li>teach others digital literacy skills through hands-on making</li>
+          <li>help people move beyond simply consuming the web to creating it</li>
+          <li>be a passionate advocate of an open and accessible web</li>
+        </ul>
+        <p><strong>Pledge now, and we'll follow up with with details about how you can teach the web!</strong></p>
+        <form className="mailinglist-signup" action={process.env.PLEDGE_MAILINGLIST_URL} method="POST" onSubmit={this.handleSubmit}>
+          <fieldset>
+            <label htmlFor="pledge-email" className="sr-only">email</label>
+            <div className="icon-field-container">
+              <i className="fa fa-envelope"></i>
+              <input id="pledge-email" name="email" type="email" size="30" placeholder="email@example.com" valueLink={this.linkState("email")} required />
+            </div>
+            <label htmlFor="pledge-pp-note" className="sr-only">I'm okay with you handling this info as you explain in your <a href="https://www.mozilla.org/en-US/privacy/websites/">privacy policy</a></label>
+            <input id="pledge-pp-note" name={process.env.PLEDGE_MAILINGLIST_PRIVACY_NAME} type="checkbox" className="sr-only" checked readOnly required />
+            <p className="pp-note">&#10003; I'm okay with you handling this info as you explain in your <a href="https://www.mozilla.org/en-US/privacy/websites/">privacy policy</a>.</p>
+            {this.renderValidationErrors()}
+          </fieldset>
+          <input type="submit" value="Submit Email" className="btn btn-awsm center-block" />
+        </form>
+      </Modal>
+    )
+  }
+});
+
+var ThankYouModal = React.createClass({
+  render: function() {
+  // we can't preset the message sharing on Facebook
+  var facebookShare = "https://www.facebook.com/sharer/sharer.php?u=" +
+                      encodeURIComponent("https://teach.mozilla.org");
+  var twitterShare = "https://twitter.com/home?status=" +
+                     encodeURIComponent("I just pledged to #TeachTheWeb! https://teach.mozilla.org");
+    return (
+      <Modal modalTitle="Thanks for your pledge!" className="modal-pledge">
+        <p>We appreciate your commitment to keeping the web open, accessible and ours.</p>
+        <p><strong>Share and tell your friends</strong></p>
+        <div className="social-share">
+          <a href={facebookShare} className="facebook">
+            <i className="fa fa-facebook"></i>
```

Comment:
I noticed the `:focus`'d but not `:hover`'d state of this link (and the twitter link too) is underlined:

![2015-06-19_8-00-51](https://cloud.githubusercontent.com/assets/124687/8253016/63e55f88-1659-11e5-8eb2-436225bc5628.jpg)

That looks a little bit funky to me, and I think the standard user agent `outline` CSS for the focused element is enough, but it's up to you really.